File: docs/PayAPP-연동API-oapi-0043/examples/SAMPLE_ANDROID/SAMPLE_ANDROID/payapp_android_sample_v3/AppLink/src/main/java/com/udid/applink/model/PARemoteReqeustResponseData.java
package com.udid.applink.model;

import android.content.Intent;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

/**
 * Create by 김진원
 */
@Getter
@Setter
@ToString(callSuper = true)
public class PARemoteReqeustResponseData extends PACommonResponseData {

    /**
     * 페이앱 결제 번호
     */
    String mul_no;
    /**
     * 결제 URL
     */
    String payurl;

    /**
     * 임의 변수 1, 2
     */
    String var1;
    String var2;


    public PARemoteReqeustResponseData(Intent data) {
        super(data);
        mul_no = data.getData().getQueryParameter("mul_no");
        payurl = data.getData().getQueryParameter("payurl");
        var1 = data.getData().getQueryParameter("var1");
        var2 = data.getData().getQueryParameter("var2");
    }
}

File: docs/PayAPP-연동API-oapi-0043/examples/SAMPLE_IOS/SAMPLE_IOS/payapp_ios_sample_v3/UniversalLink/model/PARemoteReqeustResponseData.swift
//
//  PARemoteReqeustResponseData.swift
//  Payapp_iOS_Sample_V3
//
//  Created by Jin Won Kim on 2023/06/22.
//

import UIKit

class PARemoteReqeustResponseData: PACommonResponseData {
    
    /**
     * 페이앱 결제 번호
     */
    var mul_no: String?
    
    /**
     * 결제 URL
     */
    var payurl: String?
    
    /**
     임의 변수 1, 2
     */
    var var1:String?
    var var2:String?
    
    
    override init(_ data:[String:Any]) {
        super.init(data)
        
        mul_no          = data["mul_no"] as? String
        payurl          = data["payurl"] as? String
        var1            = data["var1"] as? String
        var2            = data["var2"] as? String
    }
    
    func toString() -> String {
        var str = "state:\(state ?? ""), "
        str += "mul_no:\(mul_no ?? ""), "
        str += "payurl:\(payurl ?? ""), "
        str += "var1:\(var1 ?? ""), "
        str += "var2:\(var2 ?? "")"
        return str
    }
}
